nestkernel/event.h

```diff
@@ -1006,6 +1006,13 @@ class DataSecondaryEvent : public SecondaryEvent
     return supported_syn_ids_;
   }
 
+  /**
+   * Resets the vector of supported syn ids to those originally
+   * registered via ModelsModule or user defined Modules, i.e.,
+   * removes all syn ids created by CopyModel. This is important to
+   * maintain consistency across ResetKernel, which removes all copied
+   * models.
+   */
   void clear_supported_syn_ids()
```

Comment:
Shouldn't the method then better be called `reset_supported_syn_ids`? After all, it does not only clear, but also rebuild.

Reply:
i'm also fine with `reset_supported_syn_ids`, will change